sms.js

```diff
@@ -14,6 +15,11 @@ exports.handler = async function(event) {
     queueUrl,
     smsTemplate
   } = await getSmsConfig()
+  const sqs = new SQS({
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
```

Comment:
Running in the lambda environment the api will discover this by looking elsewhere, these shouldn't be passed.